rules-reviewed/jboss-eap/java-ee/java-ee-jaxrpc.windup.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<ruleset id="java-ee-jaxrpc" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            This ruleset provides generic migration knowledge for the Java EE specification
```

Comment:
@lincolnthree - Fair enough... we could probably get rid of the generic /javaee/ ones and collapse them all into /jboss-eap. It's not like we are going to use this to help people migrate to some other EE app server. :)